tests/test_templates.py

```diff
@@ -128,8 +127,14 @@ def test_table(self):
     </div>
 </body>
 </html>"""
-
         html = render_response(response=response, status_code=200)
-        expected = [line.strip() for line in expected.splitlines(1)]
-        html = [line.strip() for line in html.splitlines(1)]
+        expected = self._strip_ws(expected)
+        html = self._strip_ws(html)
         self.assertEqual(expected, html, "Table response incorrect")
+
+    def _strip_ws(self, in_text):
+        out_text = ""
+        for line in in_text.splitlines(1):
+            if line.strip():
+                out_text += line
+        return out_text
```

Comment:
I think we prefer a list being returned as we had before. The reason is that it's easier to work out which part of the output differs if you have a list because you get a rich response from the assert test. If you return a long string and there is a single character difference it will not be obvious when the equality test fails.